joomla/joomla-cms
# [4.0] Fix for #22070 Missing Language Key or Translation in Administrator Module

Pull Request for Issue  #22070

### Summary of Changes
Adding the missing Languge key and desk from 3.x branche

### Testing Instructions
Go to modules > administrator > mod_menu
And check if the missing desc is available

### Expected result
The missing desc should be shown!

### Actual result
![mod_menu](https://user-images.githubusercontent.com/12803406/45254783-2c94ca80-b37e-11e8-9e87-3ad8a3a67402.jpg)

### Documentation Changes Required
No
